ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -26,6 +28,8 @@ def setUp(self):
         self.article_api = ArticleAPI()
         self.test_unit_api = TestUnitAPI()
         self.contacts_api = ContactsAPI()
+        self.test_unit_page = TstUnits()
+        self.test_units_page = TstUnit()
```

Comment:
you don't used them, remove them
and if you need to them define them in test case itself for setup time save

Reply:
done